idaes/tests/test_headers.py

```diff
@@ -0,0 +1,52 @@
+"""
+Test that headers are on all files
+"""
+# stdlib
+import logging
+from pathlib import Path
+import os
+# third-party
+from addheader.add import FileFinder, detect_files
+import pytest
+import yaml
+
+_log = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def package_root():
+    """Determine package root.
+    """
+    import idaes
+    return Path(idaes.__file__).parent
+
+
+@pytest.fixture
+def patterns(package_root):
+    """Grab glob patterns from config file.
+    """
+    conf_file = package_root.parent / "addheader.cfg"
```

Comment:
As I understand the configuration file format is YAML, but the extension is `.cfg`. I don't think this is a problem (except maybe for confusing text editors/IDEs), but is there a deeper reason why the extension is not `.yml`?

Reply:
that's where the wheel stopped, not my fault. but seriously, changing to .yml